### PR TITLE
fix: exclude adobe products icons and third-party logo icons

### PR DESCRIPTION
**Pull request by jianliao**
## Description

Remove the dependency for `@spectrum/icons` and replace it with `@adobe/spectrum-css-workflow-icons`.
Since the SoT of `@adobe/spectrum-css-workflow-icons` is `@a4u/a4u-collection-react-spectrum-open-source-release` and `@a4u/a4u-collection-react-spectrum-open-source-color-icons-release`, all Adobe product logo icons and third-party logo icons are excluded.

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] This pull request is ready to merge.

### CODE REVIEW

File: components/icon/README.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
 The icons component contains all UI icons used for components, as well as CSS for workflow icons.
-
-## Updating from A4U
-
-To update from A4U:
-
-1. Update the versions of the A4U packages in `package.json`, if necessary
-1. Make sure you've executed `npm install` in the root of the project within the Adobe network so `node_modules/@spectrum/spectrum-icons` is present
-1. Run `npm run update`
-1. Commit the new files with `git add .; git commit -m "Update icons"`
```
Comment (lazd) [inline components/icon/README.md]: Yay! Glad to see this gone.

File: package.json [hunk 2]
```diff
@@ -55,8 +55,7 @@
   },
   "optionalDependencies": {
     "@a4u/a4u-collection-essential-ui-icons-release": "^4.1.0",
```
Comment (lazd) [inline package.json]: While we're at it, can we get a pubic package for these little fellas too?

Comment (jianliao) [reply package.json]: Yeah, we should create a ticket for next sprint.

**Comment by lazd** [comment]
@jianliao just checked build output, seems like there are some differences:

Before:
image

After:
image

Let's go ahead and kill all the `.html` and `sites/` folder as well.

**Comment by misterbrownlee** [comment]
Hi @jianliao could you find time to resolve the merge conflicts and close this PR?  Thanks!

**Comment by jianliao** [comment]
#662 will cover the change in this PR.

**Pull request closed without merging**